test_environments/message_and_event_brokers/kafka/sasl_scram/with_tls/scram_sha_512/README.md

```diff
@@ -0,0 +1,25 @@
+This is a simple setup of Kafka with SASL_SCRAM (SHA-512 algorithm) authentication and TLS encryption.
```

Comment:
```suggestion
# Testing Kafka broker with SASL_SCRAM (SHA-512 algorithm) and TLS

This is a simple setup of Kafka with SASL_SCRAM (SHA-512 algorithm) authentication and TLS encryption.
```

Reply:
I added `# Setup Kafka broker with SASL SCRAM (SHA-512) authentication and TLS encryption`